nautobot/extras/tests/test_filters.py

```diff
@@ -252,25 +255,25 @@ def test_site(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
     def test_role(self):
-        device_roles = DeviceRole.objects.all()[:2]
+        device_roles = self.device_roles[:2]
         params = {"role_id": [device_roles[0].pk, device_roles[1].pk]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(device_roles))
         params = {"role": [device_roles[0].slug, device_roles[1].slug]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(device_roles))
```

Comment:
It seems weird to me that these changes are needed - as written it sure looks like we should always have exactly two results here and below.

Reply:
Yeah, there were cases where it was sometimes resulting in a result of 1 so I made it where the roles being selected weren't adjacent and then relied on the count there instead of hard-coding the integer value expected. Meh.